notebooks/iris.ipynb

```diff
@@ -56,6 +56,13 @@
     "    X_scaled, y, test_size=0.2, random_state=2)"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    ax2.set_xlabel("epoch no");
</pre><p>Why not simply "epoch"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pzawistowski/notebook-review-test/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>